### PR TITLE
doc: add module support in README

### PR DESCRIPTION
**Pull request by blackrez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,23 @@ check the `examples` subdirectory.
...
+Install it via go get
 ```
 go get github.com/owulveryck/onnx-go
 ```
+Or add it in go.mod
+
+```
+module myproject.org/package
+
+go 1.12
```
Comment (owulveryck) [inline README.md]: is go 1.12 mandatory? If so, we should write it?

File: README.md [hunk 2]
```diff
@@ -35,9 +35,23 @@ check the `examples` subdirectory.
...
+
+require (
+        github.com/google/flatbuffers v1.11.0 // indirect
+        github.com/owulveryck/onnx-go v0.0.0-20190605120756-5af9985
```
Comment (owulveryck) [inline README.md]: I know nothing about modules, does this version change at each commit?

Comment (owulveryck) [reply README.md]: isn't it a way to generate it "automatically"?

Comment (blackrez) [reply README.md]: User can generate by `go mod init`, you right.
I will just add the project support go modules.

File: README.md (hunk 1 above)
Comment (blackrez) [reply README.md]: The code is tested only with go 1.12. Do we support other golang version ?

Comment (owulveryck) [reply README.md]: As you told me offline, it would be nice to support also go 1.11 as it's the default version shipped with Debian.

Comment (blackrez) [reply README.md]: I tested with go 1.11 and gomodules are disable out of the box and the support is not great.

**Pull request merged**